packages/Config/RectorConfig.php

```diff
@@ -81,7 +83,7 @@ public function enableCollectors(): void
         SimpleParameterProvider::setParameter(Option::COLLECTORS, true);
     }
 
-    public function parallel(int $seconds = 120, int $maxNumberOfProcess = 16, int $jobSize = 15): void
+    public function parallel(int $seconds = 120, int $maxNumberOfProcess = 32, int $jobSize = 20): void
```

Comment:
Let's stay with PHPStan default to avoid per-project upgrades of this file:
https://phpstan.org/config-reference#parallel-processing

Reply:
iirc, 15 is to make less memory, /cc @staabm ?

Reply:
These are designed be configured per project as differt by OS, machine and project. Also low memory is not the goal, usually speed is. Let's stay in sync with PHPStan to keep it simple.

Reply:
rector has very different usage pattern of memory.

these numbers were used to allow running rector on big projects without endless amounts of memory.

Reply:
> Also low memory is not the goal, usually speed is.

its not about "low memory". its about sucking all RAM out of a system and running into swap, which turns the system to be super slow

Reply:
@TomasVotruba I think rollback to previous config is better, since the previous version is with @staabm experience on big project and large files on his PR with max chunk per-worker set: 

- https://github.com/rectorphp/rector-src/pull/4965

also I test that `60` seconds timeout seems timeout on `CodeIgniter4` project

https://github.com/codeigniter4/CodeIgniter4/actions/runs/7462946128/job/20306475124#step:9:11

set to `120` second solve it.

Reply:
These are project specific values, there is no reason to promote them to default as we do not have data about average Rector usage nor runtime. On a different project setting higher values would lead to late feedback, etc.

Customize those values to fit the particular project.